ingest/scripts/group_segments.py

```diff
@@ -189,11 +214,15 @@ def main(
 
         for field in shared_fields:
             values = {segment: segment_metadata[group[segment]][field] for segment in group}
-            deduplicated_values = set(values.values())
-            if len(deduplicated_values) != 1:
-                msg = f"Assertion failed: values for group must be identical: {values}"
-                raise ValueError(msg)
-            row[field] = deduplicated_values.pop()
+            deduplicated_values = sorted(set(values.values()))
+            if len(deduplicated_values) > 1:
+                if field == "authors":
+                    # For authors, we accept different orders
+                    logger.info(f"Author orders differ for group {joint_key}: {values}")
+                else:
+                    msg = f"Assertion failed: values for group must be identical: {values}"
+                    raise ValueError(msg)
+            row[field] = deduplicated_values[0]
```

Comment:
Just to be sure: this is sorted such that `0` will always be the same for the same data (even if the data comes up in different orders)? (I'm not doubting it, just checking)

Reply:
Yes it is :)! Just above